run_tests.sh

```diff
@@ -232,11 +236,17 @@ case "$action" in
     list_tests
     ;;
   test)
+    RUN_UNIT=1
+    run_user_provided_tests "$@"
+    ;;
+  test-integration)
+    TESTS_DIR='./tests/integration'
+    RUN_INTEGRATION=1
```

Comment:
Why do we need this new option?
I'm able to run a single integration test with `./run_test --integration test <test_name>`

Reply:
That's true. As we discussed in the call, I hadn't noticed this option. I updated the PR with some improvements.